### PR TITLE
Create branch.json

### PR DESCRIPTION
**Pull request by harsh-branch**
Adding a source for Branch.io

### CODE REVIEW

File: sources/branch.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "BRANCH",
+  "name": "Branch",
+  "categories": ["ANALYTICS"],
+  "organization": "Branch",
```
Comment (AnuFeb) [inline sources/branch.json:5]: Organization "Branch" doesnt exist in the registry file. Organization Ids must be defined in the [organizations]
(https://github.com/googledatastudio/ds-data-registry/tree/master/organizations) registry file.

Comment (harsh-branch) [reply sources/branch.json:5]: Created the PR for the same https://github.com/googledatastudio/ds-data-registry/pull/704

**Comment by harsh-branch** [comment]
@AnuFeb Can you please help review and merge this

**Comment by harsh-branch** [comment]
@AnuFeb requesting you to review and merge this PR.

**Review by AnuFeb — APPROVED** [review]
LGTM, Thanks for Adding!

**Comment by AnuFeb** [comment]
Could you please update the branch, I will merge the change.

**Comment by harsh-branch** [comment]
Updated the branch. @AnuFeb requesting you to merge this PR.

**Comment by harsh-branch** [comment]
Done! Update again

**Pull request merged**